modules/adtrueBidAdapter.js

```diff
@@ -478,6 +482,7 @@ export const spec = {
       return;
     }
     publisherId = conf.pubId.trim();
+    zoneId = conf.zoneId.trim();
```

Comment:
It may be possible that this is set to a non string correct?

If so, this will throw a `conf.zoneId.trim is not a function`

Probably similar to pubId right above.

For example, if a publisher puts the bid params for adtrue like

```
{
   pubId: 1234,
   zoneId: 5678
}

Trying to call `.trim()` on these will throw an exception.

Unless I am missing somewhere where these are converted to strings?